### PR TITLE
add new VM spec for datavolume based root and data-disk, without WFFC

### PR DESCRIPTION
**Pull request by Rohit-PX**
**What type of PR is this?**
integration-test

**What this PR does / why we need it**:
* Adding a new VM spec that uses only datavolume based root and datadisks, that don't use WFFC
* This way co-location will always be achieved

**Does this PR change a user-facing CRD or CLI?**:
no

**Is a release note needed?**:
no

**Does this change need to be cherry-picked to a release branch?**:
no

### CODE REVIEW

File: test/integration_test/common_test.go [hunk 1]
```diff
@@ -1729,7 +1729,7 @@ func addTestModeEnvironmentVar() error {
 
 func IsCloud() bool {
 	stc, err := operator.Instance().ListStorageClusters(defaultAdminNamespace)
-	if err == nil {
+	if len(stc.Items) > 0 && err == nil {
```
Comment (pureneelesh) [inline test/integration_test/common_test.go:1732]: remove the same check on line 1734?

**Review by pureneelesh — APPROVED** [review]

File: test/integration_test/specs/kubevirt-fedora-multiple-disks-datavol-only/fedora-vm-multidisk-datavol-only.yaml [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+            chpasswd: { expire: False }
+        name: cloudinitdisk
+  dataVolumeTemplates:
+  - metadata:
+      name: fedora-root-disk-datavol-only
+    spec:
+      source:
+        pvc:
+          name: fedora-template-pvc
+          namespace: openshift-virtualization-os-images
```
Comment (pureneelesh) [inline test/integration_test/specs/kubevirt-fedora-multiple-disks-datavol-only/fedora-vm-multidisk-datavol-only.yaml:72]: Can we add another datavolume with "blank" source?
```
blank: {}
```

Comment (Rohit-PX) [reply test/integration_test/specs/kubevirt-fedora-multiple-disks-datavol-only/fedora-vm-multidisk-datavol-only.yaml:75]: Fixed.

**Comment by Rohit-PX** [comment]
```
2024/04/16 02:37:28 WARN: Skipping testrail update for this case, testID: 93196, testrun: 0
--- PASS: TestKubevirt (2142.81s)
    --- PASS: TestKubevirt/kubeVirtHypercOneLiveMigration (2139.31s)
PASS
```

**Pull request merged**